tests/conftest.py

```diff
@@ -1,9 +1,13 @@
 # -*- coding: utf-8 -*-
 from __future__ import unicode_literals
 
+import os
 import re
+import sys
 from datetime import timedelta
 
+sys.path.insert(0, os.path.abspath('..'))
```

Comment:
Give me one good reason to modify `sys.path`.

You should be able to run the tests easily using `tox -v --recreate` or right click `run yourtest.py` in any IDE.

If we ever had to modify `sys.path` in test files, then that should go to an `__init__.py` and be in a single place.